### ISSUE TITLE
Change !send reply [msgid] to !reply [msgid]

### ISSUE THREAD

**Issue by jcserv** [white-van/WVAnonBot#46]
### What problem are you facing?

!send reply [msgid] feels unnecessarily long

### What can we do to fix that problem?

Change it to be !reply [msgid]
Only downside I can think of right now is any possible code duplication between the cases, but it'd also let us remove some command parsing logic.

Another issue: users may be used to the syntax of !send reply

Few options:
1. Make a full change and make an announcement
2. Implement the shortened command while keeping the old one, and remove it in the next release so that users are acclimated
3. If they use "send reply", send a help message telling them it has changed

**Comment by CometWhoosh** [comment]
Good idea! I'm in favour of making an announcement of the change, as well as sending them a help message telling them it's changed if they use the old syntax.

**Comment by CometWhoosh** [comment]
Just thought of this: using `!reply msg` won't work if we want to send to deep-talks, since the word "reply" replaces "send-deep". The same problem will exist for other channels once we implement scalability.

**Comment by jcserv** [comment]
!reply [msgid] can work even with scalable channels, as follows:

1. (on the backend) check that the submitter has read/write permissions for that channel
2. determine the channel based on the message being replied to, and send the reply there
